### PR TITLE
Bug 572325 FLS must check license on startup

### PR DESCRIPTION
**Pull request by eparovyshnaya**
Abstract JettyActivator to supply license protection for an arbitrary jetty-based product

Signed-off-by: eparovyshnaya <elena.parovyshnaya@gmail.com>

### CODE REVIEW

File: bundles/org.eclipse.passage.lic.jface/META-INF/MANIFEST.MF [hunk 1]
```diff
@@ -2,7 +2,7 @@ Manifest-Version: 1.0
 Automatic-Module-Name: org.eclipse.passage.lic.base.ui
 Bundle-ManifestVersion: 2
 Bundle-SymbolicName: org.eclipse.passage.lic.jface
-Bundle-Version: 1.1.0.qualifier
+Bundle-Version: 1.1.100.qualifier
```
Comment (ruspl-afed) [inline bundles/org.eclipse.passage.lic.jface/META-INF/MANIFEST.MF:5]: 1.1.1

File: bundles/org.eclipse.passage.lic.jetty/src/org/eclipse/passage/lic/internal/jetty/ui/LicensedJettyActivator.java [hunk 2]
```diff
@@ -29,30 +26,26 @@
...
 import org.eclipse.passage.lic.internal.jetty.JettyHandler;
 import org.eclipse.passage.lic.internal.jetty.JettyServer;
 import org.eclipse.passage.lic.internal.net.connect.Port;
-import org.osgi.framework.Bundle;
 import org.osgi.framework.BundleActivator;
 import org.osgi.framework.BundleContext;
-import org.osgi.framework.FrameworkUtil;
 
-public final class JettyActivator implements BundleActivator {
+public abstract class LicensedJettyActivator implements BundleActivator {
```
Comment (ruspl-afed) [inline bundles/org.eclipse.passage.lic.jetty/src/org/eclipse/passage/lic/internal/jetty/ui/LicensedJettyActivator.java:35]: I don not see where it was removed from the MANIFEST.MF after convertion to `abstract`

**Review by ruspl-afed — CHANGES_REQUESTED** [review]

File: bundles/org.eclipse.passage.lic.jface/META-INF/MANIFEST.MF (hunk 1 above)
Comment (eparovyshnaya) [reply bundles/org.eclipse.passage.lic.jface/META-INF/MANIFEST.MF]: Done

**Review by ruspl-afed — APPROVED** [review]

File: bundles/org.eclipse.passage.lic.jetty/src/org/eclipse/passage/lic/internal/jetty/ui/LicensedJettyActivator.java (hunk 2 above)
Comment (eparovyshnaya) [reply bundles/org.eclipse.passage.lic.jetty/src/org/eclipse/passage/lic/internal/jetty/ui/LicensedJettyActivator.java:35]: It's `final` ancestor resides in `lbc.jetty` and is mentioned in `manifest.mf` properly

**Pull request merged**